application/views/awards/cq/index.php

```diff
@@ -2,6 +2,7 @@
     #cqmap {
 	height: calc(100vh - 430px) !important;
 	max-height: 900px !important;
+    margin-bottom: 30px !important;
```

Comment:
I think it's better to adjust the height above instead like this: 
height: calc(100vh - 460px) !important;
Can you test that instead and see if that works better?